pipelines/azure-pipelines.yml

```
@@ -44,6 +44,7 @@ extends:
         displayName: "Publish WinGet DSC Resources"
         templateContext:
           outputs:
+          # TODO: Add pipeline artifact for NET.SDK.ToolInstaller

```

```suggestion
          - output: pipelineArtifact
            displayName: 'Publish Pipeline Microsoft.DotNet.Dsc'
            targetPath: $(Build.SourcesDirectory)\resources\Microsoft.DotNet.Dsc\
            artifactName: Microsoft.DotNet.Dsc
```
Append this to the end of this list and we should be able to run the tests.